pkg/reconciler/route/route.go

```diff
@@ -125,7 +125,7 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, r *v1.Route) pkgreconcil
 		return err
 	}
 
-	if cfgmap.FromContextOrDefaults(ctx).Features.ResponsiveRevisionGC != cfgmap.Enabled {
+	if config.FromContextOrDefaults(ctx).Features.ResponsiveRevisionGC != cfgmap.Enabled {
```

Comment:
is it a bit weird that we're comparing to a constant in another package here?

Reply:
meh, actually ignore me, I think this is just How This Works since apis/config is both a store and has the features stuff 🤷